### PR TITLE
DOCSP-39791: Databases & collections

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-39791
Staging - https://deploy-preview-87--docs-cpp.netlify.app/databases-collections/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

**Review by mcmorisi — APPROVED** [review]
LGTM, nice work!

File: source/includes/databases-collections/databases-collections.cpp [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+    {
+        // Sets read settings for the "test_database" database
+        // start-database-read-settings
+        auto db = client["test_database"];
+
+        mongocxx::read_preference rp;
+        rp.mode(mongocxx::read_preference::read_mode::k_secondary);
+        mongocxx::read_concern rc;
+        rc.acknowledge_level(mongocxx::read_concern::level::k_majority)
```
Comment (kevinAlbs) [inline source/includes/databases-collections/databases-collections.cpp:93]:
```suggestion
        rc.acknowledge_level(mongocxx::read_concern::level::k_majority);
```

File: source/includes/databases-collections/databases-collections.cpp [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+    {
+        // Sets read and write settings for the "test_collection" collection
+        // start-collection-read-write-settings
+        auto coll = client["test_database"]["test_collection"];
+
+        mongocxx::read_concern rc;
+        rc.acknowledge_level(mongocxx::read_concern::level::k_local)
+        mongocxx::write_concern wc;
+        wc.acknowledge_level(mongocxx::write_concern::level::k_acknowledged)
```
Comment (kevinAlbs) [inline source/includes/databases-collections/databases-collections.cpp:108]:
```suggestion
        wc.acknowledge_level(mongocxx::write_concern::level::k_acknowledged);
```

File: source/includes/databases-collections/databases-collections.cpp [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        // end-database-read-settings
+    }
+
+    {
+        // Sets read and write settings for the "test_collection" collection
+        // start-collection-read-write-settings
+        auto coll = client["test_database"]["test_collection"];
+
+        mongocxx::read_concern rc;
+        rc.acknowledge_level(mongocxx::read_concern::level::k_local)
```
Comment (kevinAlbs) [inline source/includes/databases-collections/databases-collections.cpp:106]:
```suggestion
        rc.acknowledge_level(mongocxx::read_concern::level::k_local);
```

**Review by kevinAlbs — APPROVED** [review]
LGTM with semicolons added.

**Pull request merged**